appium/webdriver/extensions/flutter_integration/flutter_commands.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Any, Dict, Optional, Tuple, Union
+
+from appium.common.helper import encode_file_to_base64
+from appium.webdriver.extensions.flutter_integration.flutter_finder import FlutterFinder
+from appium.webdriver.extensions.flutter_integration.scroll_directions import ScrollDirection
+from appium.webdriver.webdriver import WebDriver
+from appium.webdriver.webelement import WebElement
+
+
+class FlutterCommand:
+
+    def __init__(self, driver: WebDriver) -> None:
+        self.driver = driver
+
+    # wait commands
+
+    def wait_for_visible(
+        self,
+        locator: Union[WebElement, FlutterFinder],
+        time_out: Optional[float] = None,
```

Comment:
we have timeout everywhere. Is there any specific reason to rename the var to time_out?

Reply:
It is only used in two places: wait_for_visible and wait_for_invisible. Could you clarify the issue with the time_out name? I believe it clearly indicates its purpose for the wait

Reply:
there is no issue. It's just called `timeout` (without underscore everyhere), so it would make sense to preserve this style

Reply:
Thanks! Updated :)